tests/src/core/importType.js

```diff
@@ -69,9 +69,21 @@ describe('importType(name)', function () {
   it("should return 'builtin' for additional core modules", function() {
     // without extra config, should be marked external
     expect(importType('electron', context)).to.equal('external')
+    expect(importType('@org/foobar', context)).to.equal('external')
 
     const electronContext = testContext({ 'import/core-modules': ['electron'] })
     expect(importType('electron', electronContext)).to.equal('builtin')
+
+    const scopedContext = testContext({ 'import/core-modules': ['@org/foobar'] })
```

Comment:
we should probably also support a value of `@org` here - that way you can mark an entire package scope as builtin